source/GameLoadingPanel.cpp

```diff
@@ -73,7 +73,10 @@ void GameLoadingPanel::Draw()
 	glClear(GL_COLOR_BUFFER_BIT);
 	GameData::Background().Draw(Point(), Point());
 
-	GameData::Interfaces().Get("menu background")->Draw(Information(), this);
+	// The interface can only be drawn after the data files have been loaded
+	// to avoid a race condition.
+	if(GameData::IsDataLoaded())
+		GameData::Interfaces().Get("menu background")->Draw(Information(), this);
```

Comment:
> The race here is on accessing both Interfaces() (main culprit) but also the pointer in question, yes?

Yes.

> Basically I'm interested in finding a way for the game to not be just the starfield if someone has a slow HDD.

Well, before the async PR all a user with a slow HDD would see is ... nothing, since the window wouldn't get shown until all the data was loaded. So just seeing a star field is a clear improvement already. But yeah I see your point .